README.md

```diff
@@ -1,4 +1,56 @@
-# ceph-bootstrap - CLI tool to deploy Ceph clusters <br/> [![Build Status](https://travis-ci.com/SUSE/ceph-bootstrap.svg?branch=master)](https://travis-ci.com/SUSE/ceph-bootstrap) [![codecov](https://codecov.io/gh/SUSE/ceph-bootstrap/branch/master/graph/badge.svg)](https://codecov.io/gh/SUSE/ceph-bootstrap)
+# ceph-bootstrap - Deploy Ceph clusters using cephadm<br/> [![Build Status](https://travis-ci.com/SUSE/ceph-bootstrap.svg?branch=master)](https://travis-ci.com/SUSE/ceph-bootstrap) [![codecov](https://codecov.io/gh/SUSE/ceph-bootstrap/branch/master/graph/badge.svg)](https://codecov.io/gh/SUSE/ceph-bootstrap)
 
-`ceph-bootstrap` is a CLI tool to deploy Ceph clusters (both the upstream and SUSE
-downstream versions) using Salt.
+This project is aimed at deploy [Ceph][ceph] clusters managed by
+[cephadm][cephadm] using [Salt][salt]. We are filling the missing pieces in
+order to fully manage a Ceph cluster with cephadm:
+
+- OS management (Performing OS updates, ntp, tuning)
+- Install required RPM dependencies
+- Bootstrap cephadm
+- Enhanced bootstrapping by defining roles for Salt minions
+- Migration from [DeepSea][deepsea] to cephadm
+
+# Components
+
+This repository contains two components:
+
+1. `ceph-salt-formula` is a Salt Formula using Salt Highstates to manage Ceph
+   minions.
+2. `ceph-bootstrap` is a CLI tool to manage the Salt Formula.
+
+# Architecture
+
+![](_images/architecture.png)
+
+# Setup
+
+In order to use `ceph-bootstrap`, you need a working Salt cluster.
+
+Now, install `ceph-bootstrap` on your Salt Master from the openSUSE
+repositories:
+
+```
+zypper in ceph-bootstrap
+```
+
+# Usage
+
+```
+Usage: ceph-bootstrap [OPTIONS] COMMAND [ARGS]...
```

Comment:
This will be quickly outdated. I wonder if we should/could switch to rst and Sphinx and use then things like https://pypi.org/project/sphinxcontrib-programoutput/ for that. 
Not a merge blocker. just thinking loud...

Reply:
Could be something for the future, yes

Reply:
I agree with @toabctl here. The upstream ceph project continually has lots of trouble keeping `man CLI_TOOL_FOO` aligned with the output of `CLI_TOOL_FOO --help` - in fact, it seems to be an impossible task!

Reply:
Do you have a better text?

Reply:
Sure, instead of copy-pasting the output of `ceph-bootstrap --help`, we could write:

```
# Usage

Run `ceph-bootstrap --help` or `ceph-bootstrap <command> --help` to get check the available options and description of the commands.
```

(This is taken almost verbatim from https://github.com/SUSE/sesdev BTW.)

Reply:
ok. I've added a _real_ usage.